### PR TITLE
Filter

### PR DESCRIPTION
**Pull request by parulgupta2001**
1.Delete an item from history and delete complete history functionality added.
2.Filter by category and filter by search functionality added.

### CODE REVIEW

File: src/frontend/apiCalls/history-calls.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      { video },
+      {
+        headers: {
+          authorization: token,
+        },
+      }
+    );
+    dataDispatch({ type: "HISTORY", payload: response.data.history });
+  } catch (err) {
+    console.log(err);
```
Comment (mansipadgelwar) [inline src/frontend/apiCalls/history-calls.js:29]: use console.error or toasts to show errors

File: src/frontend/components/videoCard/videoCard.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    color: var(--color-two);
+}
+
+.img_card_container:hover {
+    transform: scale(1.2);
+}
+
+.on_hover {
+    display: none;
+    /* font-size: 5rem; */
```
Comment (mansipadgelwar) [inline src/frontend/components/videoCard/videoCard.css:15]: remove if not needed

**Review by mansipadgelwar — APPROVED** [review]
The code looks good. Remove whitespaces and use prettier. You can merge your PR.

**Review by sumitk-singh — APPROVED** [review]
readable code , code blocks make the sense.
Good to merge.

**Pull request merged**